mmv1/third_party/terraform/services/lustre/resource_lustre_instance_test.go

```diff
@@ -82,17 +83,18 @@ data "google_compute_network" "lustre-network" {
 func testAccLustreInstance_update(context map[string]interface{}) string {
 	return acctest.Nprintf(`
 resource "google_lustre_instance" "instance" {
-  instance_id         = "tf-test-my-instance%{random_suffix}"
-  location            = "us-central1-a"
-  filesystem          = "testfs"
-  capacity_gib        = 18000
-  network             = data.google_compute_network.lustre-network.id
-	description         = "test-description"
-	labels              = {
+  instance_id                 = "tf-test-my-instance%{random_suffix}"
+  location                    = "us-central1-a"
+  filesystem                  = "testfs"
+  capacity_gib                = 18000
+  network                     = data.google_compute_network.lustre-network.id
+  description                 = "test-description"
+  per_unit_storage_throughput = 1000
```

Comment:
The field can be updated to a different value to test the update.

Reply:
If the update feature is not ready, you can add `immutable: true` to this new field.

Reply:
Not supported at the moment